packages/forma-36-website/gatsby-config.js

```diff
@@ -1,9 +1,10 @@
 module.exports = {
   siteMetadata: {
     title: 'Forma 36 - The Contentful Design System',
-    promoText: 'Join the Berlin Design Systems Community Q&A on November 28th',
-    promoLink: 'https://www.meetup.com/berlin-design-systems/events/266289217/',
-    promoLinkText: 'Sign up on Meetup',
+    // Uncomment the lines below to display a promotional bar at the very top of the website
+    // promoText: 'Join the Berlin Design Systems Community Q&A on November 28th',
```

Comment:
@OriginalEXE Thanks for the contribution! Currently this won't work as GraphQL can no longer access the fields - that's why the CI is currently failing. If you update it to use empty strings instead this should work:

```
promoText: '',
promoLink: '',
promoLinkText: ''
```

Reply:
Thanks for the hint. I have updated the PR.